### PR TITLE
UI Updates

### PR DESCRIPTION
**Pull request by jwrb**
This PR rectifies and updates a few things within the Electron wallet.
- Fixed: Information is now correctly formatted on the addresses page.
- Fixed: Full height backgrounds have been fixed all over the application. No more background clipping.
- Addition: The loading window can now be dragged around the screen. Text selection on it is also disabled
- Addition: The frontpage has had sparkle applied, the interface is simpler
  - _please note there are temporary transactions on this page, ready to be filled with actual transaction information from the daemon_
- Fixed: Templates have been cleaned to remove bulk
- Fixed: CSS has been updated & minified. No more whitespace / carriage returns
- Removed: Send / Receive navigation items. These are now on the main page
  - _these still exist commented out in the jade mixins, pending the design path we take_
- Fixed: Titles on tabs having a margin far too large
- Fixed: Alignment of the Balance column on the staking page

If there are any issues, please feel free to comment on this issue.

### CODE REVIEW

File: views/mixins.jade [hunk 1]
```diff
@@ -39,14 +32,15 @@ mixin appNav()
...
           span.text-md.text-white Overview
-      li
-        a.js-tab-send.padder(href='#send')
-          i.fa.fa-chevron-up.m-r-sm.text-white
-          span.text-md.text-white Send
-      li
-        a.js-tab-transaction.padder(href='#receive')
-          i.fa.fa-chevron-down.m-r-sm.text-white
-          span.text-md.text-white Receive
+      //
```
Comment (evanlucas) [inline views/mixins.jade:35]: Let's actually remove it instead of just commenting it out

File: lib/elements/wallet.js [hunk 2]
```diff
@@ -21,13 +21,18 @@ Wallet.prototype.render = function(info, accounts) {
       : 0
   }).map(address)
   return [
-    h('#receive.js-tab-address-book-content', [
+    h('#receive.bg-light', [
       h('.wrapper', [
         h('.tab-text', [
-          h('h3.text-primary', 'Receive Paycoins')
+          h('h3.text-primary.col-sm-6.m-none.no-padder.m-b-md', 'Receive Paycoins')
+        , h('a.btn.btn-dark.pull-right.col-sm-3.col-sm-offset-3', { "href": "#generateAddress"}, [
```
Comment (evanlucas) [inline lib/elements/wallet.js]: style nit: should be <= 80 columns

File: lib/elements/overview.js [hunk 3]
```diff
@@ -13,32 +13,60 @@ inherits(Overview, BaseElement)
...
+      ])
+    , h('.col-sm-12.bg-dark.lter.flex-fill.flex-row.m-none.no-padder', [
+        h('.col-sm-6.flex-fill', [
+          h('h3.m-t-xl.m-b-none.text-white.font-thin', 'RECENT TRANSACTIONS')
+        , h('.row', [
+            h('table.col-sm-12', [
+              h('tbody', [
+                h("tr", [
+                  h("td.col-sm-1.text-3x.text-center", [ h("i.fa.fa-chevron-up") ])
+                , h("td.col-sm-6.text-white.text-lg", [ "Sent Paycoin", h("br"), h("small", [ "to paycoin address" ]) ])
```
Comment (evanlucas) [inline lib/elements/overview.js:39]: style nit: line length

File: app.js [hunk 4]
```diff
@@ -41,7 +41,7 @@ function App(el, currentWindow) {
   var nav = require('./lib/nav')(this)
   nav.setup()
 
-  var sels = '#navigation a, #navigation a span, #navigation a i'
+  var sels = '#navigation a, #navigation a span, #navigation a i, #nav a, #nav a .inner, #nav a .inner i'
```
Comment (evanlucas) [inline app.js:44]: style nit: line length

**Comment by evanlucas** [comment]
Lots of lines that are really long. Other than that LGTM

**Comment by jwrb** [comment]
```
    h('#receive.bg-light', [
      h('.wrapper', [
        h('.tab-text', [
          h('h3.text-primary.col-sm-6.m-none.no-padder.m-b-md', [
            'Receive Paycoins'
          ])
        , h('a.btn.btn-dark.pull-right.col-sm-3.col-sm-offset-3',
          { "href": "#generateAddress"}, [
            h('font-thin.m-t-b-a]', [
              "Generate Address"
            ])
          ])
        ])
      , h('table.table.table-condensed', [
          h('tbody', [accounts])
        ])
      ])
    ])
```

Would this be more appropriate for this situation?

**Comment by evanlucas** [comment]
Landed in cfbbe901e0cdd90c5d3eec3fe6e2c1aa0c5bf57a .. 921c2a10df271a050e3958c091a25bc84aa89930

**Comment by evanlucas** [comment]
Thanks

**Pull request closed without merging**